api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImpl.java

```diff
@@ -58,9 +58,6 @@
 	
 	@Override
 	public Encounter get(@Nonnull String uuid) {
-		if (uuid == null) {
```

Comment:
Why did you remove this check? Note that `@Nonnull` is a hint to the programmer; it doesn't guarantee that the value here is never null (and we should have the same checks we do [in the base class](https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/impl/BaseFhirService.java)).

Reply:
@ibacher I mistook the Nonnull annotation. I will add it back to it and the base class.

Reply:
Perfectly understandable... My IDE also tells me those checks are useless, but sometimes you have to know better than your tools.